tcms/telemetry/static/telemetry/js/testing/breakdown.js

```diff
@@ -142,6 +142,7 @@ function drawChart(data, type, selector) {
     c3.generate(chartConfig);
 }
 
+// TODO: remove this in favour of the one in utils.js
```

Comment:
This belongs in a separate commit/PR

Reply:
I know, this is why I have put a TODO, rather than move it now

Reply:
Your TODO is totally unrelated to the functionality at hand. Needs to be separated.

Reply:
it is, because, before today we had this function at `breakdown.js`. 
however, now we need it at `execution-trends.js`, so I am extracting it to `utils.js`. now, it must be removed from `breakdown.js`, because we can reuse the one from `utils.js`. however, I do not want to do that change now, so I am putting a TODO, so we don't forget to remove the duplicated code (we did not have duplication until this PR)